src/components/appRouter.js

```diff
@@ -510,42 +504,50 @@ class AppRouter {
 
     authenticate(ctx, route, callback) {
```

Comment:
I can't reach wizard.

This function is called 3 times.

1st, `ServerSignIn` but `data.StartupWizardCompleted` is `false` -> navigate `wizardstart.html`.
https://github.com/jellyfin/jellyfin-web/blob/982ae411c59cb95421ea870b0dd1726662fe15d5/src/components/appRouter.js#L516
But then: https://github.com/jellyfin/jellyfin-web/blob/982ae411c59cb95421ea870b0dd1726662fe15d5/src/components/appRouter.js#L533

2nd and 3rd, `this.firstConnectionResult` is `null`.

Reply:
Hmmm it seems you're right, I must have made changes after I got it working. Do you have any suggestions?

Reply:
I haven't tried, but shouldn't we "return" from this block? 
https://github.com/jellyfin/jellyfin-web/blob/fa1483d3c0b9e860508e7e26f1fc33f2e95acffc/src/components/appRouter.js#L509-L523

Reply:
I think this has been resolved, and I also fixed a few more edge cases.